standards/code-versioning.md

```diff
@@ -2,83 +2,110 @@
 
 ## What is Git Tagging?
 
-  * [Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) is a simple, easy way to keep track of your code at a certain point in time. Some instances where this becomes useful are:
-    * Client-facing deliverables such as milestones, UAT, or production launch/release
-    * Internal-facing code at a point that you or your team wants to demarcate as important
-    * Versioning of your code (as a default, Shift3 follows SemVer)
-
-## SemVer Basics:
-  * Always have an up-to-date documented API (in our case, README.md for our projects or Wiki)
-  * Increment your versions based on a **Major.Minor.Patch** format where:
-    * **Major** changes are breaking, non-backwards compatible changes to your code base.
-    * **Minor** changes are new features or functionality that ius still backwards-compatable
-    * **Patch** changes are bug fixes or hotfixes that only address issues and don't introduce anything new.
-  * Releases should be unalterable and locked -- forever.
-  * Initial development (so most of our lives) should be treated as 0.x.x so as to indicate that there is no release as yet.
-    * Generally, any software with a major version of 0 should be treated as unstable.
+-   [Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) is a simple, easy way to keep track of your code at a certain point in time. Some instances where this becomes useful are:
+    -   Client-facing deliverables such as milestones, UAT, or production launch/release
+    -   Internal-facing code at a point that you or your team wants to demarcate as important
+    -   Versioning of your code (as a default, Shift3 follows SemVer)
+
+## [SemVer](https://semver.org/) Basics:
+
+-   Always have an up-to-date documented API (in our case, README.md for our projects or Wiki)
+-   Increment your versions based on a **Major.Minor.Patch** format where:
+
+    -   **Major** changes are breaking, non-backwards compatible changes to your code base.
+    -   **Minor** changes are new features or functionality that ius still backwards-compatable
```

Comment:
I believe `ius` should be `is`. That's a misspelling form the original material!

Reply:
great catch!